registration-processor/registration-processor-common-camel-bridge/src/test/java/io/mosip/registration/processor/camel/bridge/PauseFlowPredicateTest.java

```diff
@@ -106,14 +106,14 @@ public void testRoutePredicatePositive() throws Exception {
 		MessageDTO messageDTO = new MessageDTO();
 		messageDTO.setRid("10002100741000120201231071308");
 		Map<String, String> tags = new HashMap<>();
-		tags.put("META_INFO-META_DATA-machineId", "10074");
+		tags.put("HOTLISTED", "operator");
 		messageDTO.setTags(tags);
 		exchange.getMessage().setBody(objectMapper.writeValueAsString(messageDTO));
 		LocalDateTime dateTimeBefore = DateUtils.getUTCCurrentDateTime().plusSeconds(432000);
 		assertTrue(pauseFlowPredicate.matches(exchange));
 		LocalDateTime dateTimeAfter = DateUtils.getUTCCurrentDateTime().plusSeconds(432000);
 		JsonObject json = new JsonObject((String) exchange.getMessage().getBody());
-		assertEquals("ResumeFromBeginning", json.getString("defaultResumeAction"));
+		assertEquals("STOP_PROCESSING", json.getString("defaultResumeAction"));

```

Comment:
in match expression above stop processing is in small case, here it is caps, is that ok ?

Reply:
Fixed